google-cloud-firestore/src/test/java/com/google/cloud/firestore/WriteBatchTest.java

```diff
@@ -329,4 +336,37 @@ public void deleteDocument() throws Exception {
     CommitRequest commitRequest = commitCapture.getValue();
     assertEquals(commit(writes.toArray(new Write[] {})), commitRequest);
   }
+
+  @Test
+  public void bulkCommit() throws Exception {
+    BatchWriteResponse.Builder response = BatchWriteResponse.newBuilder();
+    response.addWriteResultsBuilder().getUpdateTimeBuilder().setSeconds(0).setNanos(1);
```

Comment:
This is technically missing one extra call to `addWriteResultsBuilder()` (with not update time).

You can also drop the call to setSeconds(0) since it is the default value.

Reply:
done.